examples/quarto-book-netlify.yaml

```
@@ -0,0 +1,49 @@
+on:
+  push:
+    branches: main
+  pull_request:
+    branches: main
+  # to be able to trigger a manual build
+  workflow_dispatch:
+
+name: Render and deploy Book to Netlify
+
+jobs:
+  build-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Quarto
+        uses: quarto-dev/quarto-actions/install-quarto@v1
+        with:
+          # To install LaTeX to build PDF book 
+          tinytex: true 
+          # uncomment below and fill to pin a version
+          # version: 0.9.105
+
+      # Install any computing dependency you may need, usually either python and Jupyter or R
+      # More specific examples will be shared on quarto-dev/quarto-actions project
+
+      - name: Render book to all format
+        # Add any command line argument needed
+        run: |
+          quarto render
+
+      - name: Deploy to Netlify
+        id: netlify-deploy
+        uses: nwtgck/actions-netlify@v1.2.3
```

Should we use `v1` instead so that we always get stable updates ? Or do you think it is better to pin a specific version ? 

I just don't want us to often update this example at each version of `nwtgck/actions-netlify`
Usually the v1 tag will always be moved to last `v1.x.x` version: https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-tags-for-release-management 


```suggestion
        uses: nwtgck/actions-netlify@v1
```

---

I'm agree. It's a good idea to get a stable update. 
I haven't known about that `v1` moving to the last patch before.

Thanks you @cderv

---

> I haven't known about the fact that v1 will move to the last patch.

I discovered that late, but it works only if the github action project follows the advice from the link I shared. And it seems that this project do as the `v1` tag is moved to last tag each time. Same with minor version tag like `v1.2` tag